catalog-info.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
     # names that might be interested in changes to the architecture of this
     # component.
     openedx.org/arch-interest-groups: "bmtcril,feanil"
+    openedx.org/release: "master"
```

Comment:
The default branch here is `main`

Reply:
@bmtcril  Sorry I have updated this, :)

Reply:
also if you can manually tag it for open-release/sumac.1, that would be great :)